package.json

```diff
@@ -15,7 +15,7 @@
     "heroku-postbuild": "make emojis && webpack -p --config ./webpack.dev.js",
     "start": "webpack-dev-server --config ./webpack.dev.js --open",
     "lint": "eslint . --ext .jsx --ext .js",
-    "pretest": "npm run lint && npm run prettier",
+    "pretest": "npm run lint",
```

Comment:
Removed Prettier from this because `npm run lint` already runs Prettier by way of our `@readme/eslint-config` shared config that uses the `eslint-plugin-prettier` package.